frmChanThreadWatch.cs

```diff
@@ -419,8 +420,12 @@ public partial class frmChanThreadWatch : Form {
                             });
                         }
                         else {
-                            Process.Start(dir);
-                        }
+							if(RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) {
+								Process.Start($"file://{Uri.EscapeUriString(dir)}"); // xdg-open fails on spaces
```

Comment:
I am not the most familiar with the GNU/Linux ecosystem, so what are the implications and possible side effects for people on `OSPlatform.Linux` but not using xdg-open?
Also, isn't removing the spaces from your download directory in the settings a more sensible approach as a workaround for this very particular problem?
Finally, is it probable that this would be fixed in an upcoming version of this command or is that expected behavior?